packages/react/src/grid/PageGrid.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { PropsWithChildren } from 'react'
+
+export type PageGridColumnNumber = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
```

Comment:
Is no being use here. but in the `GridCell` component. Would opt for declaring the type there (to keep stuff close to where it is being used) or create separate type file for it.

Reply:
My reasoning: the grid cell is asking the grid how many columns it consists of.

The type is indeed not used in the grid component, but the essence is that the number 12 is the grid’s responsibility.

I’m keeping it here.